src/iss.js

```diff
@@ -293,6 +294,35 @@ export async function request(path: string, data: ?searchRequest): Object {
     urlobj.auth = urlobj.search = urlobj.querystring = urlobj.query = null;
     url_ = url.format(urlobj);
 
+    /*
+     * Encode data into the URI ourselves
+     * until we can work around
+     * https://github.com/jedmao/iso-http/issues/2
+     * No flow until comprehensions are supported
+    */
+    var serialized = ""
+    if (data) {
+        // Flow can't tell that `data` isn't null inside a closure
+        var _data = data;
+        serialized = Object.keys(data).map(function (key) {
```

Comment:
I think the problem is that flow can't determine that the closure never leaks outside the `if` block.